osmaxx/conversion/converters/converter.py

```diff
@@ -2,7 +2,7 @@
 from osmaxx.conversion.converters import converter_gis
 from osmaxx.conversion.converters import converter_pbf
 from osmaxx.conversion.job_dispatcher.rq_dispatcher import rq_enqueue_with_settings
-from osmaxx.conversion_api.formats import FGDB, SHAPEFILE, GPKG, SPATIALITE, GARMIN, PBF
+from osmaxx.conversion.constants.formats import FGDB, SHAPEFILE, GPKG, SPATIALITE, GARMIN, PBF
```

Comment:
`from osmaxx.conversion.constants import formats` might be worth considering as an import and usage would become `formats.GARMIN`.